modules/eco-self-node-remediation-operator-about.adoc

```diff
@@ -55,10 +55,16 @@ If a watchdog device is unavailable, the `SelfNodeRemediationConfig` CR uses a s
 [id="understanding-self-node-remediation-remediation-template-config_{context}"]
 == Understanding the Self Node Remediation Template configuration
 
-The Self Node Remediation Operator also creates the `SelfNodeRemediationTemplate` CR with the name `self-node-remediation-default-template` in the Self Node Remediation Operator's namespace. This CR defines the remediation strategy for the nodes.
+The Self Node Remediation Operator also creates the `SelfNodeRemediationTemplate` CR, which creates the following remediation templates:
```

Comment:
is it not CRD ?
cc: @mshitrit

Reply:
Well... it's both CRD and CR 😉 
We have a SelfNodeRemediationTemplate CRD, and we create default 2 CRs from it.

Maybe re-order things a bit, to avoid confusion?

```
The Self Node Remediation Operator also creates the `SelfNodeRemediationTemplate` CRD. This CRD defines the remediation strategy for the nodes. The following remediation strategies are available:
[...]
The Self Node Remediation Operator creates a SelfNodeRemediationTemplate CR for each strategy with these names:
[...]
The `SelfNodeRemediationTemplate` CR resembles the following YAML file:
[...]
```

Reply:
The way I look at it is: the CRD is the blueprint on how to create the CR.
So I'd expect to have one type of CRD but there could be multiple CRs created out of this CRD.